anoynymblogging/provision.py

```diff
@@ -0,0 +1,88 @@
+"""Usage.
+
+Usage:
+  provision.py create <name> <tag>
+  provision.py list <tag>
+  provision.py delete <tag>
+
+Options:
+  -h --help     Show this screen.
+
+"""
+
+import sys
+import requests
+from docopt import docopt
+import json
+import environ
+
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, True)
+)
+# reading .env file
+environ.Env.read_env()
+
+api_token = env('DO_TOKEN')
+api_url_base = 'https://api.digitalocean.com/v2/'
+
+headers = {'Content-Type': 'application/json',
+           'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+def create(args):
+    name = args['<name>']
+    tag = args['<tag>']
+    print(name)
+    print(tag)
+    data = {
+        "name": name,
+        "region": "blr1",
+        "size": "s-1vcpu-1gb",
+        "image": "ubuntu-16-04-x64",
+        "ssh_keys": [23341811],
+        "backups": False,
+        "ipv6": True,
+        "user_data": None,
+        "private_networking": None,
+        "volumes": None,
+        "tags": [
+            "name" + tag
+        ]
+    }
+    api_url = '{0}droplets'.format(api_url_base)
```

Comment:
Use f strings from now on, replace all format usage.

Reply:
Define base as http://example.com without the slash and compute endpoint like base + '/action'. And base must be in capitals since its a constant.